### PR TITLE
Bump timeout in force_scheduled_tasks

### PR DESCRIPTION
**Pull request by jknphy**
Bump timeout to 10 minutes in force_scheduled_tasks.

- Related ticket: https://progress.opensuse.org/issues/57026
- Verification run: [sle-15-SP2-Online-ppc64le-Build126.1-RAID6@ppc64le-no-tmpfs](https://openqa.suse.de/t3810028)

### CODE REVIEW

File: tests/console/force_scheduled_tasks.pm [hunk 1]
```diff
@@ -52,7 +52,7 @@ sub run {
     my $systemd_tasks_cmd = 'echo "Triggering systemd timed service $i" && systemctl start $i';
     $systemd_tasks_cmd .= ' && systemctl mask $i.{service,timer}' unless get_var('SOFTFAIL_BSC1063638');
     assert_script_run(
-'for i in $(systemctl list-units --type=timer --state=active --no-legend | sed -e \'s/\(\S\+\)\.timer\s.*/\1/\'); do ' . $systemd_tasks_cmd . '; done', 420);
+'for i in $(systemctl list-units --type=timer --state=active --no-legend | sed -e \'s/\(\S\+\)\.timer\s.*/\1/\'); do ' . $systemd_tasks_cmd . '; done', 540);
```
Comment (foursixnine) [inline tests/console/force_scheduled_tasks.pm]: make it 10 minutes already :), after all... we want just this task to simply run...

Comment (jknphy) [reply tests/console/force_scheduled_tasks.pm]: yes, it is a round number, why not! :) Addressed.

**Comment by foursixnine** [comment]
@jknphy Just fix the last commit message :D and we're good to go!

**Comment by jknphy** [comment]
upss I merge it whit that dummy commit, sorry! wondering how can I delete it now?

**Pull request merged**